src/parser/jobs/mnk/NOTES.md

```diff
@@ -1,28 +1,43 @@
 ## how2monk good and punch other things good too
 
 ### Uptime
-- [ ] Don't clip or drop Demolish by more than 2 seconds, unless reapplying would be on GL0
+- [?] Don't clip or drop Demolish by more than 2 seconds, unless reapplying would be on GL0
 - [ ] Avoid GL0 Demolish applications, if opener or downtime ignore this
-- [ ] GL3 uptime
+- [x] GL3 uptime
+- [ ] GL3 efficient recovery
+- [x] GL3 holding with RoE
+- [ ] Advise when downtime was short enough and damage taken for RoE?
+- [x] Wasted RoE
+- [ ] GL loading with RoW (warn on WT without RoW after it)
 - [ ] General auto-attack uptime
+- [ ] Earth Tackle, not even once
+- [x] Fists, make sure at least one is up
+- [ ] Fists of Fire uptime?
 
 ### AoE
-- [ ] Make sure there's enough enemies for RB or AotD effectiveness
+- [x] Make sure there's enough enemies for RB or AotD effectiveness
 - [ ] Allow AotD for Silence if we can tell if the enemy is actually interrupted
+- [ ] Warn on RB being used as a range extender
 
 ### Tornado Kick
 - [ ] Minor on 8th RoF GCD instead of 9th
 - [ ] Major if just after RoF drops
 - [ ] Check recovery doesn't take more than 4 GCDs (PB is 3, ending before Snap and using RoW is 4)
 - [ ] Use before forced downtime longer than RoE can be used, this might be fight specific and is hard to track since FFLogs is weird about GL refreshes
+- [ ] Implement setup/recovery scenarios? Usually this is just "fix your opener"
 
 ### Positionals
 - [ ] Can kinda guess this from the damage mainly because MNK potency is high but it's a bit giri-giri
-- [ ] Crit the Boot
+- [x] Crit the Boot
```

Comment:
Derp, I committed my global notes. The positionals module currently tracks only crit Bootshines, and makes a checklist for crit rate. I'll probably remove the checklist because it's not super useful but I'm still not too sure on how effective that module is besides for obviously missed positionals, it depends a lot on fights too.

I had an idea of tracking the damage amount for diff skills and normalising (not the way parser modules normalise) the damage across buffs, and using that for positional info. MNK has significant potency differences for positional that other jobs don't, plus special effects, so it's easier to tell when we hit positionals. There's also the position of character and enemy data that could be used but it becomes fight specific to calculate position plus we'd need to know where the aggro holder is to determine direction, and that only works assuming the boss is facing that player in the first place.